containers/2_ApplicationSpecific/abaqus/README.md

```diff
@@ -0,0 +1,28 @@
+# Running Abaqus with Apptainer
+
+Abaqus 2024 is available via an Apptainer/Singularity container. You can find the Abaqus container here: `/util/software/containers/x86_64/abaqus-2024.sif`
+This is a large file (10GB) so please be sure to copy it to a location where you have enough space (i.e. your project directory), or you can run the container from this location.
+
+**Note**: Apptainer/Singularity is only available on compute and compile nodes. Please refer to our [container documentation](https://docs.ccr.buffalo.edu/en/latest/howto/containerization/) for more information on using containers.
+
+## Abaqus Command Line/Batch Script
+
+To run Abaqus from the command line, run this command: `$ apptainer exec -B /util:/util,/scratch:/scratch /util/software/containers/x86_64/abaqus-2024.sif /bin/bash`
+
+**Tip**: The `-B` option is telling Apptainer/Singularity which directories on the node you want to bind mount into the container. You will automatically get access to your home directory. The directories we have listed here to bind mount are required. If you also want access to your project directory you can add `,/projects:/projects` to the list or your specific directory `,/projects/academic/[YourGroupName]:/projects/academic/[YourGroupName]` for example.
+
+A Slurm script [`abaqus-test.sh`](./abaqus-test.sh) is provided with all necessary configuration for running Abaqus in a batch script. Update the file paths and resource requests according to your needs. Once the container starts you'll see the Apptainer command prompt and then you can run the Abaqus command as normal. For example: `Apptainer> abaqus help`
```

Comment:
The batch script needs to run unattended so the full list of command options need to be specified by the user.
```suggestion
A Slurm script [`abaqus-test.sh`](./abaqus-test.sh) is provided with all necessary configuration for running Abaqus as a batch job. Update the file paths and resource requests according to your needs and add the Abaqus options you need to use at the end of the Apptainer command line.
```